From d5fea85ab037145b33e2ce16cf3b62bc9806fcf2 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Thu, 15 Jun 2017 12:12:49 -0500
Subject: [PATCH 1901/5242] MLK-15098: ASoC: fsl: fix imx-pcm build error when
 no dma
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit  e1ded6f2655c14f651a5037154b05ac9489eaafc from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix build error when SOC_IMX_PCM_DMA is not enabled
error: expected identifier or ‘(’ before ‘{’ token})

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/imx-pcm.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/imx-pcm.h b/sound/soc/fsl/imx-pcm.h
index 0b2d775..c9c03d2 100644
--- a/sound/soc/fsl/imx-pcm.h
+++ b/sound/soc/fsl/imx-pcm.h
@@ -60,7 +60,7 @@ static inline int imx_pcm_dma_init(struct platform_device *pdev, size_t size)
 {
 	return -ENODEV;
 }
-static inline int imx_pcm_platform_register(struct device *dev);
+static inline int imx_pcm_platform_register(struct device *dev)
 {
 	return -ENODEV;
 }
-- 
1.7.9.5

